test/inductor/test_cpu_select_algorithm.py

```diff
@@ -1396,7 +1396,13 @@ def forward(self, x):
     @torch.no_grad
     @unittest.skipIf(not TEST_MKL, "Test requires MKL")
     @set_num_threads(1)
-    @parametrize("batch_size", (1024,))
+    @parametrize(
+        "batch_size",
+        (
+            512,
+            1024,
```

Comment:
Perhaps just using `512` would be good enough? I'm concerned about the added test time...

Reply:
Changed to just keep `512`.